src/openrct2/paint/Supports.cpp

```diff
@@ -1539,3 +1539,38 @@ bool PathBSupportsPaintSetup(
 
     return false; // AND
 }
+
+constexpr MetalSupportType RotatedMetalSupports[MetalSupportTypeCount][NumOrthogonalDirections] = {
```

Comment:
```suggestion
static constexpr MetalSupportType RotatedMetalSupports[MetalSupportTypeCount][NumOrthogonalDirections] = {
```
No need to have this a public symbol.